examples/vue3/vite.config.ts

```diff
@@ -29,8 +29,11 @@ export default defineConfig({
 
   server: {
     hmr: {
-      clientPort: process.env.CODESPACES ? 443 : undefined,
```

Comment:
Why this change?

Reply:
haha im not actually sure, I was trying to set it up on a different port so I could use it on my remote server and I guess I accidentally deleted that line, I will fix it tomorrow.

Reply:
I reverted the change,  I appreciate the help :)